db/initsql/dbschema.sql

```diff
@@ -0,0 +1,74 @@
+create table if not exists room
+(
+	id integer not null
+		constraint room_pkey
+			primary key,
+	name varchar(25) not null,
+	room_type varchar(6) not null
+);
+
+alter table room owner to postgres;
+
+create table if not exists users
+(
+	id integer not null
+		constraint users_pkey
+			primary key,
+	name varchar(20) not null,
+	surname varchar(20) not null
```

Comment:
może się mylę ale jak ktoś przyjdzie do nas drugi raz to jego historii nie będziemy mieli wymieszaną w masie innych użytkowników o tym samym imieniu i nazwisku? chyba, że użytkownik dostaje id i nim się posługuje?

Reply:
> może się mylę ale jak ktoś przyjdzie do nas drugi raz to jego historii nie będziemy mieli wymieszaną w masie innych użytkowników o tym samym imieniu i nazwisku? chyba, że użytkownik dostaje id i nim się posługuje?

Tak będziemy mieli wymieszaną. Generalnie zrobiłem założenia że potrzebujemy danych użytkownika tyłko dla wystawiania faktury/kupowania biłetów bez rejestracji
Można jaknajbardziej zrobić żeby użytkownik dostawał id + możliwość rejestracji i możliwość logowania przy kolejnych zakupach. Natomiast nie było takich wymagań biznesowych